components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -217,14 +217,14 @@ watch(
 onBeforeMount(() => {
   if (listingCartEnabled.value) {
     listingCartStore.clear()
-  }
 
-  fetchSearch({
-    page: startPage.value,
-    search: parseSearch(props.search),
-  }).then(() => {
-    isLoading.value = false
-  })
+    fetchSearch({
+      page: startPage.value,
+      search: parseSearch(props.search),
+    }).then(() => {
+      isLoading.value = false
+    })
+  }
 })
```

Comment:
This breaks when listingCartEnabled is false

Reply:
Made sure this doesn't affect anything else

Reply:
@daiagi Could you please confirm whether there is any side effect here?

<img width="489" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/88cd1e19-5ec8-4f19-a8a0-19dbe0bfd3ca">

Reply:
> This breaks when listingCartEnabled is false

Then why does it only happen on a few drops instead of all? This part of the code does not change recently. Is it the root reason for this bug?